### PR TITLE
Feat/4033  add self to frame src for calendars

### PR DESCRIPTION
**Pull request by asf-algoo**
#4033 

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [X] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [X] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by asf-algoo** [comment]
Failing tests are unstable tests about invalid mentions disabled on develop.

**Review by inkhey — APPROVED** [review]

File: backend/tracim_backend/applications/agenda/application.py [hunk 1]
```diff
@@ -104,6 +106,11 @@ def load_controllers(
         configurator.include(agenda_controller.bind, route_prefix=BASE_API)
         configurator.include(radicale_proxy_controller.bind)
 
+    def get_content_security_policy_directives(
+        self, app_config: CFG
+    ) -> typing.Tuple[typing.Tuple[str, str], ...]:
```
[the remark this answers is not in the document]
Comment (raphj) [reply backend/tracim_backend/applications/agenda/application.py:111]: Okay, but by the way, aren't `None` and `()` violations of this type?

[the remark this answers is not in the document]
Comment (grignards) [reply backend/tracim_backend/applications/agenda/application.py:111]: None yes, `()` is ok.

**Review by grignards — APPROVED** [review]

**Comment by grignards** [comment]
Tested on branch.

**Pull request merged**